vulcanai/models/basenetwork.py

```diff
@@ -196,17 +194,13 @@ def _set_final_layer_parameters(self, pred_activation, criter_spec):
                 criterion specification with name and all its parameters.
 
         """
-        self._final_transform = None
 
         if isinstance(criter_spec, nn.CrossEntropyLoss):
-            if pred_activation:
-                raise ValueError("The nn.CrossEntropyLoss class combines  \
+            raise ValueError("The nn.CrossEntropyLoss class combines  \
                             nn.NLLLoss and softmax for improved efficiency, \
-                            you cannot set pred_activation when \
-                            criter_spec is set to an instance of this class. \
-                            Set pred_activation to none or change criter_spec."
-                                 )
-            self._final_transform = nn.Softmax(dim=1)
+                            but is not compatible with our current \
+                            implementation of vulcan"
```

Comment:
I mean... it is compatible except that it becomes more difficult to build ONNX-exportable Vulcan models? let's try to find a better way to word this error or is there something else we can do to make it more inclusive for `CrossEntropyLoss`?

Reply:
No it's no longer compatible as it stands because I removed the softmax final_transform parameter. This is something we could add back in, up for debate. I took it out because there's limited benefit for (our) applications and I didn't want to have future oopsies.